frontend/src/views/admin/matching/applicant-view/grid/grid-view.tsx

```diff
@@ -100,6 +102,13 @@ function GridSection({
             Positions > Assignments tab."
                     />
                 )}
+                {header === "Unassignable" && (
```

Comment:
I don't like comparing against display names. Display names aren't type checked. Can you compare against the status instead?